app/utils/Process.js

```diff
@@ -110,9 +110,10 @@ const getPlugin = (fileType, algorithm) => {
 
 class Process extends EventEmitter {
 
-    constructor(file, algorithms) {
+    constructor(file, algorithms, keepSource) {
         super();
         this.file = file;
+        this.path = this._getPath(keepSource);
```

Comment:
Save the `keepSource` instead of the calculated path.

If you copy the file in the constructor, you will have a delay before the interface shows the process begin.